source/libs/sync/src/syncRequestVote.c

```diff
@@ -90,6 +91,12 @@ int32_t syncNodeOnRequestVote(SSyncNode* ths, const SRpcMsg* pRpcMsg) {
   int32_t          ret = 0;
   SyncRequestVote* pMsg = pRpcMsg->pCont;
 
+  if(CID(&(pMsg->srcId)) != CID(&(ths->myRaftId)))
```

Comment:
The checking is ok, but unnecessary, since syncNodeInRaftGroup() may filter the wrong msg too.